src/Subscriptions/Actions/LoadSubscriptionsListTableAssets.php

```diff
@@ -60,6 +63,31 @@ public function __invoke()
         );
     }
 
+    /**
+     * @unreleased
+     */
+    private function getStatuses(): array
+    {
+        $statuses = [];
+
+        foreach(SubscriptionStatus::labels() as $value => $label) {
+            if ($value !== SubscriptionStatus::ACTIVE) {
+                $statuses[] = [
+                    'value' => $value,
+                    'text' => $label,
+                ];
+            }
+        }
+
+        // Make active status default
+        return array_merge([
+            [
+                'value' => SubscriptionStatus::ACTIVE,
+                'text' => __('Active', 'give'),
+            ],
+        ], $statuses);
```

Comment:
I think the default option should display all subscriptions regardless of the status, so I would suggest replacing this logic with something like this:

```
$statuses = [];

foreach (SubscriptionStatus::labels() as $value => $label) {
    $statuses[] = [
        'value' => $value,
        'text' => $label,
    ];
}

return array_merge([
    [
        'value' => 'all',
        'text' => __('All Status', 'give'),
    ],
], $statuses);
```